src/renderers/testing/ReactTestRenderer.js

```diff
@@ -83,8 +86,11 @@ class ReactTestComponent {
 
   getPublicInstance(transaction: ReactTestReconcileTransaction): Object {
     var element = this._currentElement;
-    var options = transaction.getTestOptions();
-    return options.createNodeMock(element);
+    var options = this._hostContainerInfo;
+    if (options && options.createNodeMock) {
+      return options.createNodeMock(element);
+    }
+    return {};
```

Comment:
This was because of `flow`, `_hostContainerInfo` is typed as `null | Object`, which is valid since it is `null` when the instance is created. The `_hostContainerInfo` isn't populated until the instance mounts.

While it never *should* run be `null` when this method is called, flow doesn't know that. This method is also typed as returning an `Object`. That's easy to change though if you want to do `null | Object`.